packages/features/ee/teams/components/GoogleWorkspaceInviteButton.tsx

```diff
@@ -0,0 +1,125 @@
+import { UsersIcon, XIcon } from "lucide-react";
+import { useRouter } from "next/router";
+import { useState } from "react";
+import type { PropsWithChildren } from "react";
+
+import { useFlags } from "@calcom/features/flags/hooks";
```

Comment:
To use the cached context instead. The fetch already happens on the provider.

```suggestion
import { useFlagMap } from "@calcom/features/flags/hooks";
```